jobs/github-testers-macros.yaml

```diff
@@ -265,7 +265,7 @@
           TEST_DIR=`readlink -f test-results`
           chcon -Rt svirt_sandbox_file_t "$TEST_DIR"
           TESTS=(${{TESTS[@]}})
-          parallel -j100% sudo docker run --rm -v "$TEST_DIR":/junit "$CONTAINER" launch-test "{{}}" dnf ::: "${{TESTS[@]}}" || :
+          parallel -j100% sudo docker run --rm -v "$TEST_DIR":/junit "$CONTAINER" launch-test "{{}}" ::: dnf-2 dnf-3 ::: "${{TESTS[@]}}" || :
```

Comment:
@ignatenkobrain Here is parallel trigger, I only extended it. Therefore jenkins do parallel run!!!